src/strategies/dutchv3_strategy.rs

```diff
@@ -89,14 +95,11 @@ impl<M: Middleware + 'static> Strategy<Event, Action> for UniswapXDutchV3Fill<M>
     }
 
     // Process incoming events, seeing if we can arb new orders, and updating the internal state on new blocks.
-    async fn process_event(&mut self, event: Event) -> Vec<Action> {
+    async fn process_event(&mut self, event: Event) -> Option<Action> {
```

Comment:
seems like the artemis-core change happened again here

Reply:
This is a reversion back to what is in main
https://github.com/Uniswap/uniswapx-artemis/blob/851854185deea61f0c6591be7e3db3c70f15322f/src/strategies/uniswapx_strategy.rs#L91